docs/modules/reference/pages/provisioning/policies/node-category.adoc

```diff
@@ -1,3 +1,5 @@
 = Node Categorization Policy
+:description: Learn about the Node Categorization policy in OpenNMS Horizon/Meridian to set categories to assign to provisioned nodes.
 
-The Node Categorization Policy lets you set categories to assign to provisioned nodes.
+The Node Categorization policy lets you set categories to assign to provisioned nodes.
+//this section seems sparse
```

Comment:
It is sparse, but there's not really much to configure for it that's different from the common settings in the parent page.